main.py

```diff
@@ -72,19 +75,23 @@ def parse_args(args):
 
     # Load the data.
     logger.info('Loading data and building iterators.')
-    train_iter, val_iter, test, de_field, en_field = utils.torchtext_iterators(
+    train_iter, val_iter, test, src_field, trg_field = utils.torchtext_iterators(
+        root_data_path=args['data_path'],
+        src_language=args['src_language'],
+        trg_language=args['trg_language'],
         device=args['device'],
         batch_size=args['batch_size'],
         min_freq=args['min_freq'],
-        max_len=args['max_len']
+        max_len=args['max_len'],
+        load_fasttext_embeds=args['fasttext_embeds']
     )
 
     train_iter = [item for i, item in enumerate(train_iter) if i < 10]
     val_iter = train_iter
```

Comment:
This would be a good PR to remove this debugging code.

Reply:
Will do.